### ISSUE TITLE
Storage values are off by 10x or so?

### ISSUE THREAD

**Issue by kdayday** [NREL-SIIP/ExtremeSolarTexas#15]
I calculate the power capacity of storage as around 500 MW, instead of several GW? 
```
sum(get_input_active_power_limits(get_component(GenericBattery, system, b))[:max]
                      for b in storage_names)*100
542.4000000000001
```
 ```get_state_of_charge_limits``` and ```get_input_active_power_limits``` and ```get_output_active_power_limits``` are all returning very small values

**Comment by claytonpbarrows** [comment]
What is your system unit base setting? You can run `get_units_base(system)` to check.

**Comment by jd-lara** [comment]
The previous version of the data set had a conversion issue fixed to address #1. Since the storage units are meant to substitute very small generators that are less than 100 MW in the system then it reduced the storage installation by a significant amount. 

The total amount is way lower than the expected interconnection Queue for ERCOT, so I need to make adjustments and also increase the ratings.
